docs/pages/blog/turbo-1-2-0.mdx

```diff
@@ -0,0 +1,112 @@
+
+## New Task Filtering API
+
+We are excited to release one of our most requested features: the ability to expressively filter tasks through a `--filter` flag. The `--filter` flag is the much more powerful successor to the current combination of `--scope`, `--include-dependencies`, `--since`, and `--no-deps` flags. With `--filter` you can tell `turbo` to restrict executing commands to a subset of matched packages in your monorepo based on name, folder, and even if it has changed since a git commit ref.
+
+Take a look at some examples of what you can accomplish with the new `--filter` command:
+
+- `--filter=<package_name>` - match by exact package name or glob pattern
+- `--filter=...<package_name>`- match by package name/glob and include all dependent packages of match(es)
+- `--filter=...^<package_name>`- match by package name/glob and include all dependent packages of matches, but exclude the matches themselves
+- `--filter=<package_name>...` - match by package name/glob and include all the matched packages' dependencies
+- `--filter=<package_name>^...` - match by package name/glob and include all matched package dependencies, but exclude the matches themselves
+- `--filter={./path/to/package}` - match by path or filesystem glob pattern
+- `--filter=[origin/main]` - match by changed packages since a git commit ref
+
+You can use multiple filters together to get even more granular filtering as well as combine each part of the above patterns `{}`, `[]` , `^` , and `...` to express more complex behavior.
+
+For example, assuming you have an app located in `./apps/web` directory, some local packages that it uses as dependencies, and a Turborepo pipeline where `test` depends on `^build` [topologically](https://turborepo.org/docs/glossary#topological-order), running:
+
+```sh
+turbo run test --filter={./apps/web}[HEAD^1]^...
+```
+
+would tell `turbo` to ensure that dependencies are built and to run the `test` script in all of the local dependencies of the app located in `./apps/web`, not including that app itself, if the app has changed since HEAD^1.
+
+We believe `--filter` is more expressive and intuitive than `--scope`. In the future, we will be slowly deprecating the `--scope` flag.
+
+For more details and examples, refer to the new [filtering documentation](/docs/features/filter).
+
+## Debug and Automate with `--dry-run`
+
+You can now see the impact of `turbo run` without actually executing any commands by appending either `--dry-run` or `--dry-run=json` to any `turbo run` command. This will result in either human or JSON output.
+
+Dry runs are incredibly useful for two main situations:
+
+- Debugging and testing run options
+- Using `turbo` filtering and task graphs for building automations
+
+![Turborepo Dry Run](/images/blog/turbo-1-2-0/turbo-dry-run.png)
+
+We hope that this will improve visibility into what `turbo` is doing, speeding along debugging as well as make it easier to leverage `turbo` in dynamic CI/CD systems.
```

Comment:
```suggestion
We hope that this will improve visibility into what `turbo` is doing, speeding up debugging as well as make it easier to leverage `turbo` in dynamic CI/CD systems.
```